apps/meteor/app/api/server/v1/billing.ts

```diff
@@ -0,0 +1,33 @@
+import { getCheckoutUrl } from '../../../cloud/server/functions/getCheckoutUrl';
+import { getWorkspaceAccessTokenWithScope } from '../../../cloud/server/functions/getWorkspaceAccessTokenWithScope';
+import { getURL } from '../../../utils/server/getURL';
+import { API } from '../api';
+
+API.v1.addRoute(
+	'billing.checkoutUrl',
+	{ authRequired: true, permissionsRequired: ['manage-cloud'] },
+	{
+		async get() {
+			const { token } = await getWorkspaceAccessTokenWithScope('workspace:billing');
+			if (!token) {
+				return API.v1.unauthorized();
+			}
+
+			const manageSubscriptionURL =
+				process.env.NODE_ENV !== 'production' ? 'http://localhost:3000/admin/manage-subscription/' : getURL('admin/manage-subscription');
```

Comment:
this is actually the same, right? like, when running in development `getUrl` will return `localhost:3000`

```suggestion
			const manageSubscriptionURL = getURL('admin/manage-subscription');
```

Reply:
I thought so... but apparently not :( , because of that I was getting an error because the billing service validate the URL

Reply:
logs: 
![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/271bbe8a-be5b-4adc-9eb0-cbe22f17ccbf)